audio-worklet/design-pattern/shared-buffer/shared-buffer-worker.js

```diff
@@ -119,6 +126,15 @@ function initialize(options) {
     CONFIG.channelCount = options.channelCount;
   }
 
+  if (!self.SharedArrayBuffer) {
+    postMessage({
+      message: 'WORKER_ERROR',
+      detail: `SharedArrayBuffer is not supported. See the article above to
```

Comment:
This message doesn't really help, does it?  Where is the "article above"?  Maybe link (with shortened URL) to the actual article?

Reply:
My mistake - now the page link is ready so I can add it in here. I also prefer the actual link than the shortened one.